mantidproject/mantid
# Expose the Q summation functionality on the settings tab

**Merge BEFORE #21299**

Exposes the `ReductionType` and `SummationType` options of `ReflectometryReductionOneAuto` on the Reflectometry GUI settings tab via two comboboxes.

**To test:**
1. Open the ISIS Reflectometry GUI.
2. Open the .tbl file `INTER_NR_test2.tbl` from the ISIS Sample data. (`Reflectometry -> Import TBL`).
3. Experiment with processing individual rows with `SumInLambda` and then `SumInQ` with each ReductionType.
4. Check the corresponding `IvsQ_{RunNumber}` workspace's history (`Right Click -> Show History`) ensuring that the correct `ReductionType` and `SummationType` were passed to `ReflectometryReductionOneAuto`. 
5. Ensure that the ReductionType box is enabled **iff** the SummationType is set to `SumInQ`.



Fixes #21353. 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
